quality/BUILD.bazel

```
@@ -62,6 +63,8 @@ licence_test(
         "sw/otbn/crypto/rsa_verify.s",
         "sw/otbn/crypto/handwritten/rsa_verify_3072.s",
         "sw/otbn/crypto/sha512.s",
+        # Code based on tiny_sha3
+        "sw/otbn/crypto/symmetric.s",
```

Looks like the license checker is failing in CI because this file name has changed, so the exception no longer triggers:
```suggestion
        "sw/otbn/crypto/sha3-shake.s",
```